packages/jsActions/nanoflow-actions-native/src/client/Reload.ts

```diff
@@ -0,0 +1,18 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+/**
+ * @returns {Promise.<void>}
+ */
+export async function reload(): Promise<void> {
+    // BEGIN USER CODE
+    mx.reload();
+    return new Promise(() => {
+        /** never resolve **/
```

Comment:
I did not understand the need for this at first. I'm glad I found the info in the MR conversations.

It would help future developers understand why a perpetual Promise is needed by adding a comment to the code describing the reasoning.

Reply:
Indeed, this is a good idea! 

Added!

Reply:
Thnx